portal/src/views/shared/Tiptap.vue

```diff
@@ -70,11 +70,6 @@ export default Vue.extend({
                 this.editor.setOptions({ editable: !value });
             }
         },
-        value(value: string): void {
```

Comment:
@jlewallen do you remember if this code block should do anything in particular? It seems that adding/editing a comment works fine without it, and it fixes the issue from the ticket.